src/miniscript/context.rs

```diff
@@ -271,6 +271,9 @@ pub trait ScriptContext:
     fn is_tapctx() -> bool {
         return false;
     }
+
+    /// Local helper function to display error messages with context
+    fn to_string() -> String;
```

Comment:
We shouldn't have our own method for this. We should implement the `fmt::Display` trait and require it for `Ctx` objects.

Reply:
These are not created, so I don't know if this approach can work. Right now, I can use `Ctx::to_string()` without a need for `self.`

Reply:
Ah, I see!